packages/shared.net.fetch/src/createFetchOrThrow.ts

```diff
@@ -69,7 +71,17 @@ export function createFetchOrThrow(fetchFn: typeof fetch = fetch) {
 function convertError(
   e: any,
   msgIfNotError: string = "An unknown error occurred",
+  response?: Response,
 ) {
+  if (response?.status && response.statusText) {
+    // Catches errors like 429 where there is no response body but a code is returned
+    return new UnknownError(
```

Comment:
Hmm. I don't think a statusText is a requirement. I'd probably not do this separate if statement and instead just include the statusCode on the 2 throws below. If you think the status text is important you could add it to the first argument of the message down there.

The use of `UNKNOWN` aligns with the existing palantir api errors that are used by conjure and co and that populate through so using a custom message like the `statusText` for that feels off.